packages/nextjs/components/ExtensionCard.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import { usePlausible } from "next-plausible";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+
+type Extension = {
+  name: string;
+  description: string;
+  github: string;
+  installCommand: string;
+  builder: string;
+  coBuilders: string[];
+  youtube?: string;
+};
+
+export const ExtensionCard = ({ extension, isCurated }: { extension: Extension; isCurated: boolean }) => {
+  const [commandCopied, setCommandCopied] = useState(false);
+  const plausible = usePlausible();
+
+  const handleInstallClick = () => {
+    navigator.clipboard.writeText(extension.installCommand);
+    setCommandCopied(true);
+    setTimeout(() => {
+      setCommandCopied(false);
+    }, 800);
+
+    const githubRepo = isCurated
+      ? `Curated/${extension.name}`
+      : extension.github.split("github.com/")[1] || extension.github;
+
+    // Track the click event with GitHub repo as a prop
+    plausible("extensionCopyClick", { props: { id: githubRepo } });
+  };
+
+  return (
```

Comment:
![image](https://github.com/user-attachments/assets/a9cf4843-7c44-49eb-bf9b-31ccaa9ce508)

How about we position the copy command at the bottom of the card? (no matter how large is the description)

Reply:
Agree! It looks more coherent. Also added a bit more spacing between cards and gave the install command box full width, I think it looks better.

![image](https://github.com/user-attachments/assets/ef3eefe8-eac9-41c9-b9cd-8c6dd40cd298)